### DISCUSSION TITLE
How can I get the accuracies in different labels respectively?

### DISCUSSION THREAD

**Discussion by ZiyingHuang** [open-mmlab/mmaction2#2781] [Q&A] [answered]
How can I get the accuracies in different labels respectively? A line of the work_dirs is as follow:
01/16 09:04:25 - mmengine - INFO - Epoch(val) [35][4/4]    acc/top1: 0.8850  acc/top5: 1.0000  acc/mean1: 0.8403  data_time: 2.4839  time: 2.5480

I think the item " acc/top1" is the average accuracy of all the data set. There are many videos in some labels, but some are few. The  average accuracy can't make sense. So I want to get the accuracies in different labels respectively. How can I realize it?

**Comment by rlleshi** [comment, marked as answer]
You could calculate it yourself given the testing results or try to compute a confusion matrix using the script [here](https://github.com/open-mmlab/mmaction2/blob/main/tools/analysis_tools/confusion_matrix.py).

**Reply by ZiyingHuang** [reply]
Thanks!

**Reply by ZiyingHuang** [reply]
May I inquire about if I want to know, which samples were not recognized correctly and what categories they were identified as. How can I get it?

**Reply by rlleshi** [reply]
You can just save the test outputs when you test your model with _tools/test.py_ using the _--dump_ flag, and that analyze that file.
